plugins/gamecenter/game_center.mm

```diff
@@ -336,7 +336,7 @@
 			ret["signature"] = [[signature base64EncodedStringWithOptions:0] UTF8String];
 			ret["salt"] = [[salt base64EncodedStringWithOptions:0] UTF8String];
 			ret["timestamp"] = timestamp;
-			if (@available(iOS 13, *)) {
+			if (@available(iOS 13.5, *)) {
```

Comment:
Any reason to change this part? 
[Docs](https://developer.apple.com/documentation/gamekit/gkplayer/3174857-teamplayerid?language=objc) says it's available even at lower than 13 iOS version. It's actually `playerID` that got deprecated at 13 version. 
And having `generateIdentityVerificationSignatureWithCompletionHandler` deprecated at 13.5 shouldn't affect that part.
This should probably stay as it is.

Reply:
`fetchItemsForIdentityVerificationSignature` available in 13.5+ and `teamPlayerID` can't be verify at lower version. If return `teamPlayerID` you get signature error on server-side.

Reply:
~~Maybe using [`gamePlayerID`](https://developer.apple.com/documentation/gamekit/gkplayer/3113960-gameplayerid?language=objc) would be better then? Since `playerID` is deprecated at 13.0, I'm not sure it's a good idea to leave it hanging.~~
Found [this thread](https://developer.apple.com/forums/thread/117617) seems like there is no way around that issue.